### PR TITLE
Implemented a functional dropdown. State in sync with buttons.

### PR DESCRIPTION
**Pull request by Kenneth-Funes-TU-Delft**
- Implemented a AnnotationTypeDropdown based on the antd dropdown. Notes: there is a good likelihood that this can be trimmed and used as a direct instance given the experience with it. Also, this needs to be reusable, so we may simply fallback into using "Dropdown" directly.
- Tried to set the styling for the dropdown, making a layout with a sidebar on the left and the video on the right. However, it is currently not working
 as it seems the video is being forced to render ocupying all available
space. This may be forced somewhere in a higher level.
- Added another task instance in the journey, for simple testing.

### CODE REVIEW

File: samples/action_annotation/action_annotation.py [hunk 1]
```diff
@@ -5,9 +5,10 @@
 config.load_environment("local")
 
 my_task_1 = tasks.ActionAnnotationTaskSpec(name="My Task 1", input=None, media=None)
+my_task_2 = tasks.ActionAnnotationTaskSpec(name="My Task 1", input=None, media=None)
```
Comment (Era-Dorta-TU-Delft) [inline samples/action_annotation/action_annotation.py:8]: Why are you adding the second task here? As I understand, we only need one task.

File: covfee/client/tasks/action_annotation/index.tsx [hunk 2]
```diff
@@ -42,38 +47,47 @@ const ActionAnnotationTask: React.FC<Props> = (props) => {
...
-        </button>
+      <div className={styles.action_annotation_task}>
+        <div className={styles.sidebar}>
+          <AnnotationTypeDropdown
+            annotation_types={annotation_types}
+            itemClick={(item_key: string) =>
+              dispatch(actions.setActiveAnnotation(item_key))
+            }
+            selected_annotation={active_annotation}
+          ></AnnotationTypeDropdown>
```
Comment (Era-Dorta-TU-Delft) [inline covfee/client/tasks/action_annotation/index.tsx:58]: I think this can be simplified as
```suggestion
          >
```

**Review by Era-Dorta-TU-Delft — APPROVED** [review]
Looks good to me so far.

File: samples/action_annotation/action_annotation.py (hunk 1 above)
Comment (Kenneth-Funes-TU-Delft) [reply samples/action_annotation/action_annotation.py:8]: This was just experimentation on my end, to see what happens with journey as there are more tasks. It's also useful to validate proper state separation between the two instances of the same task.

I suggest to keep for now as it does not interfere, but of course, we will intuitively remove it once launching the actual annotation campaign.

**Pull request merged**